### ISSUE TITLE
New check: Development Status Chain

### ISSUE THREAD

**Issue by max3903** [OCA/pylint-odoo#221]
Check that modules at development status N only depend on modules at status >= N:

* Beta module depends on Beta, Stable or Mature modules
* Stable module  depends on Stable or Mature modules
* Mature module depends on Mature modules

Depends on #199

**Comment by moylop260** [comment]
We have a problem with this validation.
Currently, lint is not cloning other modules.
We have available the modules of the main project cloned.
Then, we couldn't check it from a static checker like pylint.
Maybe we could create a small script from MQT directly.

**Comment by moylop260** [comment]
Won't fix from pylint-odoo
